pallets/msa/src/rpc/src/tests/mod.rs

```diff
@@ -16,6 +16,7 @@ const SCHEMA_FOR_A: u16 = 1;
 const SCHEMA_FOR_A_AND_B: u16 = 2;
 const SCHEMA_FOR_B: u16 = 3;
 const NOT_EXIST_MSA: u64 = 100;
+const BLOCK_ZERO: BlockNumber = 0;
```

Comment:
We could possibly move the definition for 
`const BLOCK_ZERO: BlockNumber = 0;`
to `common/primitives/src/msa.rs` and import it here as well as `other_tests.rs`

Reply:
Ultimately I would like to see a `tests` module exported from `common`, but I feel that's out of scope here. And I don't want to introduce a new dependency between `pallet-msa` and `pallet-msa-rpc` that doesn't currently exist, so I'm just going to keep BLOCK_ZERO local in both crates.

Reply:
Ah, fair enough. I was thinking of a common declaration somewhere since its used by both the msa rpc tests and general unit tests